src/components/Dialog/Dialog.tsx

```diff
@@ -65,7 +65,6 @@ export class Dialog extends React.Component<IDialogProps, any> {
             ariaDescribedBy={ subText ? id + '-subText' : '' }
             onDismiss={ onDismiss }
             >
-            <div className={ dialogClassName }>
```

Comment:
This is was the cause that we saw two ms-Dialog. It is already there at: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/src/components/Dialog/Dialog.tsx#L62